bedrock/mozorg/templates/mozorg/rise25/landing.html

```diff
@@ -195,66 +114,32 @@ <h3 class="r25-c-category-title">Change Agents</h3>
           AI environment of equality and empowerment.
         </p>
       </div>
-      {% if switch('rise25-nominations-open') %}
-        <p class="r25-c-category-cta">
-          <a class="r25-c-button" href="https://mozilla.formstack.com/forms/rise25" rel="external noopener" target="_blank">Nominate</a>
-        </p>
-      {% endif %}
+
+      <p class="r25-c-category-cta">
+        <a class="r25-c-button" href="https://rise25.mozilla.org/change-agents/" rel="external noopener" target="_blank">See the honorees</a>
+      </p>
     </div>
-  </section>
 
-  <section class="mzp-l-content mzp-t-content-md">
-    <div class="r25-blog">
-      <div class="r25-blog-body">
-        <h3 class="r25-blog-title">Who’s shaping the future of AI? We’re on the hunt.</h3>
-        <p class="r25-blog-cta">
-          <a href="https://blog.mozilla.org/en/mozilla/mozilla-rise25-ai-nominations/{{ utm_params }}" rel="external noopener" target="_blank">Read our blog post</a>
+    <div id="entrepreneurs" class="r25-c-category r25-t-purple">
+      <div class="r25-c-category-body">
+        <h3 class="r25-c-category-title">Entrepreneurs</h3>
+        <p class="r25-c-category-desc">Fueling AI’s evolution with visionary ventures.</p>
+        <p class="r25-c-category-expand">
+          These daring individuals are transforming imaginative ideas into reality.
+          They’re crafting businesses and solutions with AI to meet societal needs,
+          improve everyday life and forge new technological paths. They embody
+          innovation, steering startups and projects with a commitment to ethical
+          standards, inclusiveness and enhancing human welfare through technology.
         </p>
       </div>
 
-      {{ resp_img(
-        url='img/mozorg/rise25/blog-thumbnail.jpg',
-        srcset={
-          'img/mozorg/rise25/blog-thumbnail-high-res.jpg': '1.5x'
-        },
-        optional_attributes={
-          'width': '230',
-          'height': '153',
-          'class': 'r25-blog-image'
-        }
-      ) }}
-    </div>
-
-    <div class="r25-spread-secondary">
-      <h3>Help us spread the word!</h3>
-      {% include "mozorg/rise25/includes/social-share.html" %}
```

Comment:
Both occurrences of this include are removed, the include itself can be deleted too now?

Reply:
doh, good catch!